src/scripts/main.js

```diff
@@ -1 +1,69 @@
 'use strict';
+'use strict';
+
+const logo = document.querySelector('.logo');
+
+const firstPromise = new Promise((resolve, reject) => {
+  logo?.addEventListener('click', (e) => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => reject(Error('First promise was rejected!')), 3000);
+});
+
+const leftClickProm = new Promise((resolve, reject) => {
+  logo?.addEventListener('click', (e) => {
+    resolve('promise was resolved');
+  });
+});
+const rightClickProm = new Promise((resolve, reject) => {
+  logo?.addEventListener('contextmenu', (e) => {
+    resolve('promise was resolved');
+  });
+});
+
+const secondPromise = Promise.any([leftClickProm, rightClickProm]);
+
+const thirdPromise = Promise.all([leftClickProm, rightClickProm]);
+
+firstPromise
+  .then((result) => {
+    document.body.insertAdjacentHTML(
+      'beforeend',
+      '<div data-qa="notification" class="success">First Promise resolved</div>',
```

Comment:
Fix lint error:
![image](https://github.com/user-attachments/assets/a4c5009a-82c6-46f0-a883-3ca18bf63904)

Write open, close tags and content on separate lines